### DISCUSSION TITLE
figmaEX add-on not working on macOS Ventura 13.5

### DISCUSSION THREAD

**Discussion by cherejigheorghe** [Moonvy/figmaEX#31] [Q&A]
Hi,

I found this amazing add-on for Figma, downloaded it and installed it but it is not working in Figma app on my MacBook Air M1.
I have enabled the figmaEX add-on in System Settings > App Management. Please check the screenshot below: 
<img width="706" alt="image" src="https://github.com/Moonvy/figmaEX/assets/60112647/8fc4da34-56a9-4d71-becb-b40b8c62b650">

Did anyone encounter this issue before? Is there any way I can fix it? 
PS. I tried uninstalling and installing the add-on again. I did that with Figma app as well but still without succes.

Thanks in advance, 
Gheorghe

**Comment by yArna** [comment]
https://github.com/Moonvy/figmaEX/issues/21
